packages/blade/src/components/Alert/StyledAlert.d.ts

```diff
@@ -0,0 +1,13 @@
+import type { ReactNode } from 'react';
+
+import type { IntentTypes } from '~tokens/theme/theme';
+
+export type StyledAlertProps = {
+  children: ReactNode[];
+  contrastType: 'lowContrast' | 'highContrast';
+  intent: Exclude<IntentTypes, 'neutral'>;
```

Comment:
Why neutral is excluded here?

Reply:
`neutral` isn't a part of intents on [Alert](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=7021%3A0)